modules/cnv-updating-access-mode-for-live-migration.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * cnv/live_migration/cnv-live-migration.adoc
+
+[id="cnv-updating-access-mode-for-live-migration_{context}"]
+= Updating access mode for LiveMigration
+
+Default storage classes use a ReadWriteOnce (RWO) access mode by default.
```

Comment:
Storage classes do NOT define RWO or RWX, storage classes define reclaimPolicy and volumeBinding modes. Persistent Volumes define which accessModes they support. If we have a dynamic provisioner that creates PVs, then the provisioners configuration will determine if it can create them. For manually created PVs the yaml/json of the PV will determine what access modes it supports.

The accessMode is one of the criteria kubernetes uses to determine if it can match a PVC with a PV. So if a PVC requests RWX, but no PVs with RWX are available it will not match them.

In short this has NOTHING to do with storage classes.